ui/index.js

```diff
@@ -29,6 +29,8 @@ import { _setBackgroundConnection } from './store/action-queue';
 
 log.setLevel(global.METAMASK_DEBUG ? 'debug' : 'warn');
 
+const STORE_KEY = 'flatmap-v3';
```

Comment:
This is irrelevant to the test, i was just fooling around with storage and wanted a key to increment for reading and setting to see what happened.